cypress/integration/plugins/security-dashboards-plugin/change_tenant_successfully.js

```diff
@@ -0,0 +1,65 @@
+/*
+
+* Copyright OpenSearch Contributors
+
+* SPDX-License-Identifier: Apache-2.0
+
+*/
+
+import { CURRENT_TENANT } from '../../../utils/commands';
+import tenantDescription from '../../../fixtures/plugins/security-dashboards-plugin/tenants/testTenant.json';
+
+const tenantName = 'test';
+
+if (Cypress.env('SECURITY_ENABLED')) {
+  describe('Switch tenants when visiting copyed links: ', () => {
+    before(() => {
+      cy.server();
+
+      cy.createTenant(tenantName, tenantDescription);
+    });
+    if (Cypress.browser.name === 'chrome') {
```

Comment:
Curious, why is this limited to chrome?

Reply:
That was erroneously left after the testing phase, it has now been removed.